### PR TITLE
Add ability for tracer to incorporate KMLs generated by external server

### PR DESCRIPTION
**Pull request by BenjaminPelletier**
This PR adds a `KML_SERVER` flag to the `tracer` tool which `tracer` will send new logs to in order to generate KMLs representing the state of the system.  Those KMLs are cached by `tracer` and then served to clients when requested.  If the new flag is not specified, `tracer` behaves as before.

### CODE REVIEW

File: monitoring/tracer/run_poll_locally.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+docker build \
+    -f monitoring/tracer/Dockerfile \
+    -t interuss/dss/tracer \
+    --build-arg version=`scripts/git/commit.sh` \
+    monitoring
+
+docker run --name tracer_poll \
+  --rm \
+  -e TRACER_OPTIONS="${TRACER_OPTIONS}" \
+  -v /Users/pelletierb/Documents/test/localmock:/logs \
```
Comment (pratibhagupta2109) [inline monitoring/tracer/run_poll_locally.sh:25]: Hardcoded path.

File: monitoring/tracer/run_subscribe_locally.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    -f monitoring/tracer/Dockerfile \
+    -t interuss/dss/tracer \
+    --build-arg version=`scripts/git/commit.sh` \
+    monitoring
+
+docker run --name tracer_subscribe \
+  --rm \
+  -e TRACER_OPTIONS="${TRACER_OPTIONS}" \
+  -p ${PORT}:5000 \
+  -v /Users/pelletierb/Documents/test/localmock:/logs \
```
Comment (pratibhagupta2109) [inline monitoring/tracer/run_subscribe_locally.sh:27]: ditto

**Review by pratibhagupta2109 — APPROVED** [review]

File: monitoring/tracer/run_poll_locally.sh (hunk 1 above)
Comment (BenjaminPelletier) [reply monitoring/tracer/run_poll_locally.sh:25]: Good catch, thanks!

File: monitoring/tracer/run_subscribe_locally.sh (hunk 2 above)
Comment (BenjaminPelletier) [reply monitoring/tracer/run_subscribe_locally.sh:27]: Good catch, thanks!

**Pull request merged**